src/elements/FunctionsTableRow/FunctionsTableRow.js

```diff
@@ -1,53 +1,157 @@
-import React from 'react'
+import React, { useRef } from 'react'
 import PropTypes from 'prop-types'
+import lodash from 'lodash'
```

Comment:
If you're using only `.isEqual` method then please import only it:
```js
import { isEqual } from 'lodash'
```
If you import the entire lodash library, please use the classic `_` variable name:
```js
import _ from 'lodash';
```
Then use:
```js
_.method()
```